tests/dag/config/base/bigquery.yml

```diff
@@ -0,0 +1,10 @@
+type: bigquery
+method: service-account
+project: example-project
+dataset: example-dataset
+threads: 1
+keyfile: /var/keyfile.json
+timeout_seconds: 150
+priority: interactive
+location: us-west1
+retries: 1
```

Comment:
nb: newline (is there formatter/linter in this project?)

Reply:
We have linter and formatter for Python (`isort`, `black` and `flake8`), but not for YAML files. Apart from that, those yamls are just for test purposes.